examples/wearable/UIComponents/contents/presentation-views/list/main/sub-text/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta content="width=device-width, initial-scale=1, user-scalable=no" name="viewport" />
+	<title>
+		List
+	</title>
+	<link href="../../../../../lib/tau/wearable/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../../../../lib/tau/wearable/theme/default/tau.circle.css" rel="stylesheet" />
+	<link href="../../../../../css/style.css" rel="stylesheet" />
+	<link href="../../../../../css/style.circle.css" rel="stylesheet" />
+	<script src="../../../../../js/circle-helper.js"></script>
+	<script src="../../../../../lib/tau/wearable/js/tau.js"></script>
+</head>
+
+<body>
+	<div class="ui-page" data-go-to-top-button="true">
+		<header>
+			<h2 class="ui-title">
+				Sub Text
+			</h2>
+		</header>
+		<div class="ui-content">
+			<ul class="ui-listview">
+				<li>
+					<a href="sub-text.html">
+						Basic
+					</a>
```

Comment:
Please also restore comment that more examples are needed. Without such comment, this looks wired to have a sub menu for one item only.

Reply:
done